Features/Context/PlatformUI.php

```diff
@@ -408,33 +466,30 @@ protected function closeEditView()
         } catch (\Exception $e) {
         }
     }
+
     /**
-     * Attaches a file to a input field on the HTML.
+     * Returns an array with the properties contexts,
+     * if the properties use the Context Annotation.
      *
-     * @param   string $file        file name relative to mink definitions
-     * @param   string $selector    CSS file upload element selector
+     * @return array array of methods and their service dependencies
      */
-    protected function attachFile($fileName, $selector)
+    private function parseAnnotations($annotations)
```

Comment:
name should be more descriptive of actual use.
Since it parses only properties...